code/modules/projectiles/guns/launcher/bows/arrow.dm

```diff
@@ -31,38 +34,46 @@
 	S.dropInto(loc)
 	qdel(src)
 
-/obj/item/bow_ammo/arrow
-	name = "arrow"
+/obj/item/stack/material/bow_ammo/arrow
+	name = "arrows"
```

Comment:
this makes it so that the recipe name is `arrowss`, should this be `arrow` just like how bolts are `bolt`?